torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -412,18 +558,22 @@ void GpuLower::lower() {
       active_gpu_lower = nullptr;
     }
   } lower_guard(this);
+  // Copy fusion into a new kernel for processing
+  kernel_ = std::make_unique<kir::Kernel>(fusion);
+  // Alias the fusion kernel caries around as a view of itself.
+  fusion_ = kernel_->as<Fusion>();
```

Comment:
nit: No need to have `as<Fusion>()`.

Reply:
Pushing all cleanup instances of this to https://github.com/csarofeen/pytorch/pull/1390